DashSync/shared/Models/CoinJoin/DSCoinJoinWrapper.h

```diff
@@ -16,39 +16,26 @@
 //
 
 #import <Foundation/Foundation.h>
-#import "DSChain.h"
-#import "DSTransactionOutput.h"
-#import "DSCoinControl.h"
-#import "DSCompactTallyItem.h"
-#import "DSCoinJoinManager.h"
 #import "DSMasternodeGroup.h"
+#import "DSChainManager.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
+@class DSCoinJoinManager;
```

Comment:
`DSCoinJoinManager` and `DSCoinJoinWrapper` are switched as it makes more sense to keep FFI callbacks in the wrapper.